### DISCUSSION TITLE
plz tell me the new etcd cluster make using etcd snapshot.

### DISCUSSION THREAD

**Discussion by kimjinju730** [etcd-io/etcd#16170] [Q&A]
Hello, 
I've been thinking about it for a few days now, trying it out, and I'm really confused, so I'm going to ask a question.

etcd cluster
etcd1 / etcd2 / etcd3 

However, as the etcd1 and etcd2 servers died, it became **less than quorum.** 
But I can't use the etcd1 and etcd2 servers right away.

We need to rebuild the etcd cluster from the **creating snapshots** by adding new servers named etcd4 and etcd5.
Is this the right way to do it?
🙏

**Comment by jmhbnz** [comment]
Hey @kimjinju730 - Please refer to our cluster recovery guide here in the first instance: https://etcd.io/docs/v3.5/op-guide/recovery/#restoring-a-cluster

**Reply by kimjinju730** [reply]
Hey, @jmhbnz 
I know that the link you mentioned is only possible if the initial initial-cluster member is the same.

What I want is a way to restore the snapshot data to another initial-cluster member with the existing snapshot information. Please let me know if you know.

Here is simple step test senario.

1) etcd cluster member (initial cluster member)
- ETCD_INITIAL_CLUSTER=patroni-etcd-test1=http://1.1.1.1:2380, patroni-etcd-test2=http://2.2.2.2:2380, patroni-etcd-test3=http://3.3.3.3:2380

2) etcd quroum failed -> only patroni-etcd-test1=http://1.1.1.1:2380 remain server (patroni-etcd-test2/patroni-etcd-test3 server dead) 

3) I want to add the patroni-etcd-test4 server to restore the data to the old snapshot file.

My etcd version v 3.4.18
